src/cityreader/cityreader.py

```diff
@@ -1,28 +1,46 @@
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, lat and lon (representing latitude and longitude).
+import csv
 
 
-# We have a collection of US cities with population over 750,000 stored in the
-# file "cities.csv". (CSV stands for "comma-separated values".)
-#
-# In the body of the `cityreader` function, use Python's built-in "csv" module 
-# to read this file so that each record is imported into a City instance. Then
-# return the list with all the City instances from the function.
-# Google "python 3 csv" for references and use your Google-fu for other examples.
-#
-# Store the instances in the "cities" list, below.
-#
-# Note that the first line of the CSV is header that describes the fields--this
-# should not be loaded into a City object.
+class City:
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
```

Comment:
defining lat and lon as floats would be better practice as we know that lan and lon will always be floats. Just a tip for declaring Class constructors